app/models/potential_match.rb

```diff
@@ -167,7 +167,15 @@ def self.matching_signup_ids(collection, signup, collection_tag_sets, required_t
                             pluck(:challenge_signup_id).compact
 
       # now add on "any" matches for the required types
-      condition = "any_#{required_types.first.downcase} = 1"
+      condition = {
+        fandom: "any_fandom = 1",
+        character: "any_character = 1",
+        rating: "any_tating = 1",
+        relationship: "any_relationship = 1",
+        category: "any_category = 1",
+        warning: "any_warning = 1",
+        freeform: "any_freeform = 1"
+      }[ required_types.first.downcase.to_sym ]
```

Comment:
I'm not sure why Hound isn't saying anything, but I don't think we want spaces inside the brackets here.